### ISSUE TITLE
"New Features in WCAG 2.2" missing 2.5.5 Target Size (Enhanced)?

### ISSUE THREAD

**Issue by anevins12** [w3c/wcag#3882]
Is the ["New Features in WCAG 2.2" ](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2)section missing some success criteria? For example, Target Size Enhanced is not listed.

**Comment by patrickhlauke** [comment]
Target Size (Enhanced) - née just "Target Size" - was introduced in WCAG 2.1
